utils/src/test/java/org/web3j/utils/StringsTest.java

```diff
@@ -19,16 +21,11 @@
 
     @Test
     public void testToCsv() {
-        assertThat(toCsv(Collections.<String>emptyList()), is(""));
+        assertThat(toCsv(Collections.emptyList()), is(""));
         assertThat(toCsv(Collections.singletonList("a")), is("a"));
         assertThat(toCsv(Arrays.asList("a", "b", "c")), is("a, b, c"));
     }
-
-    @Test
-    public void testJoin() {
```

Comment:
How come you removed this test?

Reply:
That test was removed because I removed the Strings.join method since Java has a static String.join() method https://docs.oracle.com/javase/8/docs/api/java/lang/String.html#join-java.lang.CharSequence-java.lang.CharSequence...-  but again, if this is targetting JDK 1.6, then... whoops, undo.

Reply:
@conor10 : do I need to put back in the hand-rolled Strings.join() utility method to support JDK 1.6 ?

Reply:
Please leave the test in.

You can update the join util to be:

```java
    public static String join(List<String> src, String delimiter) {
        return src == null ? null : String.join(delimiter, src.toArray(new String[0]));
    }
```

Reply:
Will do.

When I checked usages, the only place the code referenced that method is in the `Strings` utility class itself.  The `toCsv()` method was calling it.